packages/frontend-shared/src/gql-components/promo/Promo.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="promo-box flex flex-col items-center border-gray-100 rounded-lg w-full max-w-[1004px] m-auto">
```

Comment:
Need border class to set the border width of 1px
```suggestion
  <div class="promo-box flex flex-col items-center border border-gray-100 rounded-lg w-full max-w-[1004px] m-auto">
```